src/couch_views/src/couch_views_indexer.erl

```diff
@@ -39,7 +39,8 @@ init() ->
     #{
         <<"db_name">> := DbName,
         <<"ddoc_id">> := DDocId,
-        <<"sig">> := JobSig
+        <<"sig">> := JobSig,
+        <<"retries">> := Retries
```

Comment:
If we wanted to not break existing views, asserting the retries exist, maybe we can do a 

```
maps:get(<<"retries">>, Data, retry_limit())
```